srcalc/src/main/java/gov/va/med/srcalc/domain/model/AbstractNumericalVariable.java

```diff
@@ -10,7 +12,7 @@
      */
     public static final int UNITS_MAX = 40;
     
-    private NumericalRange fRange = new NumericalRange(0.0f, true, Float.POSITIVE_INFINITY, true);
+    private NumericalRange fRange = new NumericalRange(0.0f, true, 200.0f, true);
```

Comment:
This number conflicts with the comment on getMinValue() about 100 being the default value.